src/batched/KokkosBatched_SolveUTV_TeamVector_Internal.hpp

```diff
@@ -133,6 +133,7 @@ namespace KokkosBatched {
 		   B, bs0, bs1,
 		   zero,
 		   W, ws0, ws1);
+	member.team_barrier();
```

Comment:
No barriers or reductions at the end of Gemm, so team_barrier() aded since W is immediately reused in Trsm that follows.